### ISSUE TITLE
error in definition of function kappa

### ISSUE THREAD

**Issue by guittcam** [DaluS/GEMMES#160]
in `def_function`, replace `kappalinConst + kappalinConst * pi` with `kappalinSlope=0: kappalinConst + kappalinSlope * pi`

![image](https://user-images.githubusercontent.com/86613124/156245089-e7b0e8a7-50d2-47a7-8f5b-2b7fc333788b.png)

**Comment by guittcam** [comment]
issue with the definition of shareholding in `def_functions`: replace `np.clip(GDP *` with `GDP * np.clip(` 

![image](https://user-images.githubusercontent.com/86613124/156255680-70f70a64-deda-4d5a-89d1-1c81b080d0bd.png)

**Comment by DaluS** [comment]
Yes, should be closed in your PR ! The function was not used for the moment, thanks for its testing :)
